### PR TITLE
Grounded UI - Remove Flying Dutchman

### PR DESCRIPTION
**Pull request by mphammer**

### CODE REVIEW

**Comment by operator-r2d2** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/919#" title="Author self-approved">mphammer</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=blackducksoftware%2Fsynopsys-operator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/blackducksoftware/synopsys-operator/blob/release-2019.10.x/OWNERS)~~ [mphammer]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: pkg/synopsysctl/cmd_serve_UI.go [hunk 1]
```diff
@@ -0,0 +1,652 @@
...
+		if err != nil {
+			log.Fatal(err)
+		}
+
+		// Get Polaris from secret
+		namespace = string(reqBody)
+		polarisObj, err := getPolarisFromSecret()
+		if err != nil {
+			log.Errorf("error getting Polaris: %s", err)
+			emptyResonse, _ := json.Marshal(PolarisUIRequestResponse{})
```
Comment (yashbhutwala) [inline pkg/synopsysctl/cmd_serve_UI.go]: maybe returning 404 not found is a better approach?....the contract being back-end and front-end is usually api status codes

File: pkg/synopsysctl/cmd_serve_UI.go [hunk 2]
```diff
@@ -0,0 +1,652 @@
...
+	}
+
+	polarisObj.OrganizationDetails.OrganizationProvisionOrganizationDescription = polarisUIRequestConfig.OrganizationDescription
+	polarisObj.OrganizationDetails.OrganizationProvisionOrganizationName = polarisUIRequestConfig.OrganizationName
+	polarisObj.OrganizationDetails.OrganizationProvisionAdminName = polarisUIRequestConfig.OrganizationAdminName
+	polarisObj.OrganizationDetails.OrganizationProvisionAdminUsername = polarisUIRequestConfig.OrganizationAdminUsername
+	polarisObj.OrganizationDetails.OrganizationProvisionAdminEmail = polarisUIRequestConfig.OrganizationAdminEmail
+	data, err := util.ReadFileData(polarisUIRequestConfig.CoverityLicensePath)
+	if err != nil {
+		return nil, fmt.Errorf("failed to read coverity license from file: %+v", err)
```
Comment (yashbhutwala) [inline pkg/synopsysctl/cmd_serve_UI.go]: probably should validate file format, etc.

File: pkg/synopsysctl/cmd_serve_UI.go [hunk 3]
```diff
@@ -0,0 +1,652 @@
...
+
+	polarisUIRequestConfig.EnableReporting = polarisObj.EnableReporting
+	polarisUIRequestConfig.ReportStorageSize = polarisObj.ReportingSpec.ReportStorageDetails.Storage.StorageSize
+
+	polarisUIRequestConfig.OrganizationDescription = polarisObj.OrganizationDetails.OrganizationProvisionOrganizationDescription
+	polarisUIRequestConfig.OrganizationName = polarisObj.OrganizationDetails.OrganizationProvisionOrganizationName
+	polarisUIRequestConfig.OrganizationAdminName = polarisObj.OrganizationDetails.OrganizationProvisionAdminName
+	polarisUIRequestConfig.OrganizationAdminUsername = polarisObj.OrganizationDetails.OrganizationProvisionAdminUsername
+	polarisUIRequestConfig.OrganizationAdminEmail = polarisObj.OrganizationDetails.OrganizationProvisionAdminEmail
+	polarisUIRequestConfig.CoverityLicensePath = "" // User needs to provide License Path every time
```
Comment (yashbhutwala) [inline pkg/synopsysctl/cmd_serve_UI.go]: fine for now, probably can make it so if valid license is already in the secret, maybe don't require to put the path in again.

**Comment by operator-r2d2** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/919#" title="Author self-approved">mphammer</a>*, *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/919#pullrequestreview-295213566" title="Approved">yashbhutwala</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=blackducksoftware%2Fsynopsys-operator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/blackducksoftware/synopsys-operator/blob/release-2019.10.x/OWNERS)~~ [mphammer,yashbhutwala]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by operator-r2d2** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/919#" title="Author self-approved">mphammer</a>*, *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/919#issuecomment-536733058" title="LGTM">yashbhutwala</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=blackducksoftware%2Fsynopsys-operator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/blackducksoftware/synopsys-operator/blob/release-2019.10.x/OWNERS)~~ [mphammer,yashbhutwala]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**